### PR TITLE
check alt text, captions, etc for errors. reviewed all project descri…

### PR DESCRIPTION
**Pull request by eread-usgs**
…ptions and contact info

### CODE REVIEW

**Comment by eread-usgs** [comment]
for #153

File: data/gaTable.yaml [hunk 1]
```diff
@@ -58,10 +58,11 @@
   websiteUrl: http://cida.usgs.gov/ngwmn
   botFilteringEnabled: ''
   description: >-
-    The NGWMN data portal provides on-demand access to groundwater data
+    The National Ground Water Monitoring Network Data Portal (NGWMN) data portal provides 
+	on-demand access to groundwater data
```
Comment (jiwalker-usgs) [inline data/gaTable.yaml]: tabs = bad

File: data/gaTable.yaml [hunk 2]
```diff
@@ -448,8 +449,9 @@
   websiteUrl: http://owi.usgs.gov/vizlab
   botFilteringEnabled: ''
   description: >-
-    The VizLab function rapidly develops and deploys visualizations on current
-    water issues and/or USGS Water publications or datasets. The VizLab team works with USGS
+    The Office of Water Information VizLab function rapidly develops and deploys visualizations on current
+    water issues and/or USGS Water publications or datasets. The Vizlab website is a landing page that
+	features a catalog of VizLab products. The VizLab team works with USGS
```
Comment (jiwalker-usgs) [inline data/gaTable.yaml]: here is another tab, and there are quite a few more

**Comment by eread-usgs** [comment]
ok i fixed the darn tab thing, only thing im not sure of is if the spaces are going to appear correctly now between words. @jiwalker-usgs

**Comment by jiwalker-usgs** [comment]
It should be fine, the `>-` thing clips all whitespace on the lines down to a single space.  Some of the other multi-line operators do different stuff, so it is worth worrying about.

**Pull request merged**